src/components/pages/observation/list/views/stats/table-totals.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, SimpleGrid, Text } from "@chakra-ui/react";
+import styled from "@emotion/styled";
+import React from "react";
+
+const CardLink = styled.a`
+  div {
+    border: 2px solid var(--gray-200);
+    .count {
+      font-size: 1.6rem;
+    }
+  }
+`;
```

Comment:
I guess we can just use `<Box/>` and apply style and do the same with `.count` as well

Reply:
removed CardLink